### PR TITLE
Alchemy genesis final touches

### PR DESCRIPTION
**Pull request by brodeur**

### CODE REVIEW

File: src/components/Proposal/Proposal.scss [hunk 1]
```diff
@@ -324,11 +324,22 @@ button.disabled {
     float: right;
     border: none;
     margin-top: 13px;
-    border: none;
     height: 20px;
     line-height: 20px;
 }
 
+.detailView .proposalInfo .voteStatus.hasVoted .voteRecord {
+	display: inline-block;
```
Comment (tibetsprague) [inline src/components/Proposal/Proposal.scss]: there is weird indentation all throughout the CSS, make sure you dont have tabs turned on

File: src/components/Proposal/RedemptionsString.tsx [hunk 2]
```diff
@@ -61,11 +55,13 @@ export default class RedemptionsString extends React.Component<IProps, null> {
...
 
     if (reputation.gt(zero)) {
-      rewardComponents.push(<ReputationView reputation={reputation} totalReputation={dao.reputationTotalSupply} daoName={dao.name} />);
+      rewardComponents.push(
+        <ReputationView reputation={reputation} totalReputation={dao.reputationTotalSupply} daoName={dao.name} />);
     }
 
     return <span className={css.redemptionString}>
     {rewardComponents.reduce((acc: any, v: any) => {
+      // TODO: why this "null?" test that will never pass?
```
Comment (tibetsprague) [inline src/components/Proposal/RedemptionsString.tsx]: its the initial state, we pass in null at first to set things up

File: src/components/ViewDao/DaoSidebar.tsx [hunk 3]
```diff
@@ -55,7 +55,9 @@ class DaoSidebarComponent extends React.Component<IProps, null> {
               </li>
               <li>
                 <NavLink activeClassName={css.selected} to={"/dao/" + dao.address + "/redemptions/"}>
+{/*                TODO: Make this notification work!
```
Comment (tibetsprague) [inline src/components/ViewDao/DaoSidebar.tsx]: is there an issue for this?

File: src/components/Proposal/RedemptionsString.tsx (hunk 2 above)
Comment (tibetsprague) [reply src/components/Proposal/RedemptionsString.tsx]: can we remove this comment?

Comment (brodeur) [reply src/components/Proposal/RedemptionsString.tsx]: I think so

File: src/components/Proposal/Proposal.scss (hunk 1 above)
Comment (brodeur) [reply src/components/Proposal/Proposal.scss]: OK. I did restructure the CSS a bit, but I think Sublime just uses whatever scheme is present. "detect_indentation": true...

File: src/components/ViewDao/DaoSidebar.tsx (hunk 3 above)
Comment (brodeur) [reply src/components/ViewDao/DaoSidebar.tsx]: https://github.com/daostack/alchemy/issues/572

**Comment by brodeur** [comment]
OK I pushed to remote alchemy-genesis-final-touches but can't figure out how to update the PR?

File: src/components/Proposal/RedemptionsString.tsx (hunk 2 above)
Comment (jellegerbrandy) [reply src/components/Proposal/RedemptionsString.tsx]: sorry misread the code

**Pull request merged**